src/sourcemaps/tools/webpack.ts

```diff
@@ -47,21 +89,297 @@ export const configureWebPackPlugin: SourceMapUploadToolConfigurationFunction =
       ),
     });
 
-    clack.log.step(
-      `Add the following code to your ${chalk.bold('webpack.config.js')} file:`,
-    );
+    const webpackConfigPath =
+      findFile(path.resolve(process.cwd(), 'webpack.config')) ??
+      (await askForToolConfigPath('Webpack', 'webpack.config.js'));
 
-    // Intentially logging directly to console here so that the code can be copied/pasted directly
-    // eslint-disable-next-line no-console
-    console.log(getCodeSnippet(options));
+    const successfullyAdded = webpackConfigPath
+      ? await modifyWebpackConfig(webpackConfigPath, options)
```

Comment:
l/m: Should we log something along the lines of "Modified your config. Please double check whether it makes sense?" when we modified the config?

Reply:
good idea. I think it makes sense in both cases (modified and added) because if we added a new file, users have to most likely adjust their current way of building.